convert2rhel/subscription.py

```diff
@@ -247,8 +247,15 @@ def install_rhel_subscription_manager():
     else:
         loggerinst.info("Packages installed:\n%s" % "\n".join(rpms_to_install))
         pkg_names = get_installed_submgr_pkg_names(rpms_to_install)
-        utils.changed_pkgs_control.track_installed_pkgs(pkg_names)
+        if "python3-syspurpose" in pkg_names:
+            loggerinst.debug(
+                "Untracking python3-syspurpose from installed packages"
+                "as it could be a system package installed before this process."
+            )
+            pkg_names.remove("python3-syspurpose")
```

Comment:
I'm wondering, should we check beforehand if the packages to remove are already installed on the system? Maybe with `rpm -q python3-syspurpose`? 

If we find out that the mentioned package is already present in the system before installing `subscription-manager`, what should we do? Leave it as it is? Proceed with the reinstallation of the package for the correct version for `subscription-manager`?